controllers/registration.go

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/server/web"
+	"github.com/hoangmirs/go-scraper/forms"
+)
+
+type Registration struct {
+	baseController
+}
+
+func (c *Registration) Get() {
+	web.ReadFromRequest(&c.Controller)
+
+	c.Layout = "layouts/authentication.html"
+	c.TplName = "registration/new.html"
+
+	c.Data["Title"] = "Create a new account"
+}
+
+func (c *Registration) Post() {
+	userForm := forms.RegistrationForm{}
```

Comment:
A good call to externalize this business logic to another module. Let's keep controllers thin. But the variable name is a bit confusion, shouldn't it be called `registrationForm`?

```suggestion
	registrationForm := forms.RegistrationForm{}
```

Reply:
Yes, it should